packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.component.tsx

```diff
@@ -36,24 +36,13 @@ const VitalsAndBiometricForms: React.FC<DefaultWorkspaceProps> = ({ patientUuid,
   const isTablet = useLayoutType() === 'tablet';
   const session = useSessionUser();
   const config = useConfig() as ConfigObject;
-  const { mutate } = useSWRConfig();
+  const { cache, mutate }: { cache: any; mutate: Function } = useSWRConfig();
```

Comment:
Weird that this isn't typed. Are we missing a `@types` dependency?

Reply:
No, we do have @types. 

`cache` here gets typed to `Cache<Data>`, which I presume is a wrapper around `Map`. I couldn't get `Map.prototype.keys()` to work with it, however.